scripts/Invoke-SasApprovedSoftwareAcceptance.ps1

```
@@ -0,0 +1,570 @@
+        catch { return $false }
+    }
+
+    $applicationRequired = $ProcessNames.Count -gt 0
+    $observedProcesses = @()
+    foreach ($configuredName in @($ProcessNames)) {
+        foreach ($process in @(Get-Process -Name $configuredName -ErrorAction SilentlyContinue)) {
+            $path = $null
+            $startTime = $null
+            $responding = $false
+            $windowTitle = ''
+            try { $path = $process.Path } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)